dev_guide/builds/triggering_builds.adoc

```diff
@@ -278,7 +278,13 @@ $ oc describe bc <name>
 ----
 
 If the above command does not display any webhook URLs, then no webhook trigger
-is defined for that build configuration.
+is defined for that build configuration. In order to define a new webhook trigger,
+use `oc set triggers` with a `--from` flag. For example, to create a GitHub
+webhook, use `--from-github`:
+
+----
+$ oc set triggers bc <name> --from-github
+----
```

Comment:
per comments above, break the instruction on how to add triggers out to its own section.